pulp_2to3_migration/app/migrations/0013_fix_file_remote_migration.py

```diff
@@ -16,8 +16,11 @@ def fix_file_remote(apps, schema_editor):
         remotes_pk_to_delete = []
         for importer in pulp2isoimporter_qs:
             pulp2_config = importer.pulp2_config
-            if pulp2_config.get('feed'):
-                pulp2_config['feed'] = urljoin(pulp2_config['feed'], 'PULP_MANIFEST')
+            feed = pulp2_config.get('feed')
```

Comment:
@goosemania  i have seen cases when existing migration was updated instead of writing a new one, can you remind me the details?

Reply:
What do you mean by `existing migration`?

Reply:
ah I think I got you, the django data migration.

Reply:
We update it so new users who upgrade from older version do not run into a problem.
And we add new one, so the current users could fix their setup.

Reply:
new users won't benefit from this updated migration since they will have no data at this point.
i should just write a separate migration that will take care for existing installations.